### ISSUE TITLE
Thousands of 404's from /socket.io/jsonp-polling/

### ISSUE THREAD

**Issue by aaronjwood** [socketio/socket.io#277]
In my daily server logs, I am seeing thousands of 404's to /socket.io/jsonp-polling/{somenumberhere}/0 where "somenumberhere" is a long sequence of numbers (maybe the client ID?)

I have everything set up correctly and the application that is using socket.io is working fine. Is it normal to get almost 3000 404's from URL's like I mentioned above?

If it helps, here is the application I'm working on that's using socket.io: http://aaronjwood.com:9000/
Basically, it tracks the top 10 connections to a site in real time. It's built using NodeJS so I'm using socket.io through that
